src/core_plugins/kibana/public/dashboard/dashboard.html

```diff
@@ -28,7 +28,6 @@
           <div class="kuiLocalSearch">
             <input
               parse-query
-              input-focus
```

Comment:
To do that I think we'll need to store the user's choice in local or session storage, which we don't have a great way to do in a consistent manner in Kibana at the moment. 

The only issues I have with putting a help link in the dropdown added by https://github.com/elastic/kibana/pull/10976 is that 1. it will be kind of buried and 2. this help text doesn't relate to the rest of the content of the dropdown. The dropdown manages filters while this help text is about the query bar.

It sounds like the help should be hidden by default, but easy to find. Should we put a help link in or near the query bar? 

Or, we could go back to the original implementation that hid the help after the user starts typing. I think there was significantly less risk of that becoming annoying since it wasn't in the user's face any time the query bar had focus.